### PR TITLE
Extensions module. Step 2

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/appshell/qml/DevTools/DevToolsPage.qml [hunk 1]
```diff
@@ -94,7 +94,7 @@ DockPage {
         }
     ]
 
-    central: settingsComp
+    central: extensionsComp
```
Comment (Eism) [inline src/appshell/qml/DevTools/DevToolsPage.qml:97]: Did you forget to reset this change?

File: src/framework/extensions/extensionstypes.h [hunk 2]
```diff
@@ -61,6 +50,20 @@ static inline std::string typeToString(const Type& type)
...
+{
+
+"uri": String,                    // Example: musescore://module/target/name
+"type": String,                   // Values: form
+"title": String,                  //
+"apiversion": String              // Optional default 1
+"enabled": Boolean,               // Optional default true
+"visible": Boolean,               // Optional default true
+
+"qmlFile": String                 // Path (name) of qml file
```
Comment (Eism) [inline src/framework/extensions/extensionstypes.h:64]: I think we need to clarify here: 
qmlFilePath: Relative path of qml file

File: src/appshell/qml/DevTools/DevToolsPage.qml (hunk 1 above)
Comment (igorkorsukov) [reply src/appshell/qml/DevTools/DevToolsPage.qml:97]: For now I constantly need this page, when I finish the work completely, I can return the settings..

File: src/framework/extensions/extensionstypes.h (hunk 2 above)
Comment (igorkorsukov) [reply src/framework/extensions/extensionstypes.h:64]: ok, I'll fix it in the next PR

**Review by Eism — APPROVED** [review]

**Pull request merged**